notification-channel-config-service-impl/src/main/java/org/hypertrace/notification/config/service/NotificationChannelConfigServiceImpl.java

```diff
@@ -78,7 +79,10 @@ public void updateNotificationChannel(
                           NotificationChannel.newBuilder()
                               .setId(request.getId())
                               .setNotificationChannelMutableData(
-                                  getIdPopulatedData(request.getNotificationChannelMutableData()))
+                                  utils.getHeaderPopulatedData(
```

Comment:
Not sure what this means - please rename. Also if this is the only invocation, likely makes sense to move the call to the other util internal.